.ci/scripts/clean-docker.sh

```diff
@@ -9,7 +9,7 @@ set -euxo pipefail
 # Build and test the app using the install and test make goals.
 #
 
-readonly VERSION="8.0.0-SNAPSHOT"
+readonly VERSION="$(cat $(pwd)/.stack-version)"
```

Comment:
This implementation depends on the folder you launch the script, it must be the root of the repo otherwise do not work. There is an alternative but the `.stack-version` should be moved to the `.ci/scripts` folder that I am not sure is a good idea.

```
srcdir=$(dirname "$0")
test -z "$srcdir" && srcdir=.
readonly VERSION="$(cat ${dirname}/.stack-version)"
```